installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -3,7 +3,6 @@
 securityGroupIds=$1
 subnetIds=$2
 iamRoleARN=$3
-region=$4
```

Comment:
Region should be there.

Reply:
Included region

Reply:
why, its not being used. shellcheck throws an error

Reply:
correct. you can remove it.

Reply:
region=$4
^-- SC2034: region appears unused. Verify it or export it.